qa/automation/CareerPortal.Tests.Scenarios/2. Footer links/Links.feature

```diff
@@ -0,0 +1,41 @@
+Feature: 1. Footer links
+As a user, I want to have access to contacts and additional information
+In order to get acquainted, contact the company
+
+Background: 
+ Given user is on a page on URL "https://career.quantori.com/ru"
+ And user rewinds to the footer
+
+
+Scenario: 1.1.7. Mail icon
+    When user clicks on the Mail icon
+	Then user is redirected to the mail application
+	And user sees that a message is automatically created 
+	And email is sent to career@quantori.com
+
+
+Scenario: 1.1.9. Data Cookies Settings link
+    When user clicks on the Data Cookies Settings link
+	Then user sees Cookie Notification Messages
+	And user sees automatically filled checkbox "Necessary cookies"
+	And user sees checkbox "Other cookies"
+	And user sees "Submit Preferences" button
```

Comment:
1. There is no "Data Cookies Settings" link. There is "Cookies Settings link"
2. Cookie Notification is not a message, it is a pop-up window with text and checkbox options
3. See is not an active action or state. I guess it should be replaced.